test-manf.json

```diff
@@ -23,12 +23,26 @@
       "system": "https://covid-19-diagnostics.jrc.ec.europa.eu/devices",
       "version": "2021-05-10 20:07:30 CET"
     },
+    "2108": {
+      "display": "AESKU.DIAGNOSTICS GmbH & Co. KG, AESKU.RAPID SARS-CoV-2",
+      "lang": "en",
+      "active": true,
+      "system": "https://covid-19-diagnostics.jrc.ec.europa.eu/devices",
+      "version": "2021-06-18 12:48:10 CET"
```

Comment:
CET or CEST?
Do we want UTC here instead anyway?

Reply:
The script is taking date time information from JRC database. Removal date time from the common list is recorded without time information (00:00:00 CET) and stored in the version element. Version information for Items on a common list (i.e. ACTIVE items) holds information about last update (not information when the item was included in the common list). To summarize, Version of ACTIVE items is just a version label (not date time information), while in case of INACTIVE items, it includes date of removal from the common list. I don't see the need to convert it ti UTC.

Reply:
Agreed is just a label, so we don't * need * UTC (there was already the discussion a while back about UTC <> CET):
so AFAICS 
EITHER
1. yes is "just" a version label, but I can guarantee that with both time and timezone information in there (and the time, BTW, is not zero here but a valid time e.g. 12:48:10), it will be converted
2. We may as well get CET or CEST correct - even if it is just a label
3. CE(S)T: fine for EU alone but makes less sense - even as a label - but if you start to look at it from the perpsective of the rest of the world outside of the EU then UTC would be the most neutral formulation here 
OR
1. you really do drop the time completely and just have the date - as you say the JRC source has zero for time, so why introduce the time component here? 

> "Version of ACTIVE items is just a version label (not date time information), while in case of INACTIVE items, it includes date of removal from the common list."

and notably:

> includes date of removal from the common list

thus date and not (date-)time of removal

So why are we introducing a time component suddenly when it is not there in the original? Drop the whole time and timezone component and just leave date would be one simple solution.

Reply:
Good point Gaby. 
However I think that we should take into consideration two things:
1. Can JRC modify the same element more than once in 24 hours?
2. Can we run the script more than once in 24 hours?

If the answers are positive (and I think so) than it is safer to keep date and time in the version. A simple solution would be to remove time zone information from the version label (presuming it is always CET). What do you think?

Reply:
> Can JRC modify the same element more than once in 24 hours?

I already raised this point in my email 2021-06-29 @ 14:27, which I will re-quote here for your ease of reference:


> > 2. If time part set to zero then date only - however this does mean we are limited to one update per calendar day, which may or may not turn out to be an issue in practice - but that's a problem JRC will have to solve should it arise.

The question:

> Can we run the script more than once in 24 hours?

If JRC only modify once a day, then pointless to (1) run it any more often than once per 24 hours, and (2) have any time stamp in there as the original resolution is only one day - adding our own timestamp means we become more precise but less accurate

Reply:
JRC database includes date and time of last update of each element in the common list -> we can use it as a version label, and because JRC is using one time zone only, we can remove timezone info. 
JRC can update the list more than once in 24 hours!
 
Information about removal from (or insertion to) common list does not include time of removal (it is always 00:00:00). Unfortunately date of last update is not set (updated) when item is removed, so we cannot use it.
Thus I think that It will be safer to use time information for the item version label (the script can be run more than once a day). 

So, my conclusion is to either:
1) use date and time for active as well as inactive items and lets remove time zone information
2) use only date, presuming that probability JRC will modify the same item more than once per 24 hour is minimal

@gabywh What do you think?

Reply:
Well I went for approve and merge so I guess I agree with your changes - thanks very much for the work, appreciated :)

But to at least reply to your comments:

1. If we go for time we really should specify a timezone - no timezone is implicit UTC (at least, is what most people will expect - "princiiple of least surprises") - ISO8601 compliant however you could also mark UTC as +00:00 to be explicit about no timezone offset etc. Comes down to: if we specify time then always give a timezone (or offset from UTC) with it

2. agree probability is low more than once in any 24 hour period - and even then you could argue it would be a simple patch fix for one value or another, which we could happily pick up in the next 24-hour scheduled update anyway